packages/framework-integration-tests/integration/end-to-end/subscriptions.integration.ts

```diff
@@ -115,6 +116,80 @@ describe('subscriptions', () => {
       await expect(Promise.all([promisify(observableOne), promisify(observableTwo)])).to.eventually.be.fulfilled
     })
   })
+
+  describe('with filters', () => {
+    let client: DisconnectableApolloClient
+    before(async () => {
+      client = await graphQLClientWithSubscriptions()
+    })
+    after(() => {
+      client.disconnect()
+    })
+
+    it('get a carts with a specific ID', async () => {
+      const cartID = random.uuid()
+
+      const originalSubscriptionsCount = await countSubscriptionsItems()
+      // Let's create two subscriptions to the same read model
+      const observable = cartFilteredSubscription(client, { id: { eq: cartID } })
```

Comment:
So, this is working then? Even with complex filters?

Reply:
Yes, it works even better than the queries because you can filter subscriptions by ID of property inside an array! 🎉